addons/mass_mailing/views/mailing_list_views.xml

```diff
@@ -192,6 +193,93 @@
                                 </div>
                             </div>
                         </div>
+                        <!-- Card when the Kanban view is not grouped -->
+                        <div class="oe_kanban_global_click o_mailing_list_kanban_ungrouped w-100 m-0 d-flex flex-row justify-content-between align-items-center flex-wrap">
+                            <div class="col-lg-2 col-sm-6 col-11 py-0 my-auto">
+                                <div class="d-flex text-large">
+                                    <span class="d-inline-block text-truncate font-weight-normal text-large"
+                                          t-att-title="record.name.value">
+                                        <field name="name"/>
+                                    </span>
+                                </div>
+                            </div>
+                            <div class="o_mailing_list_kanban_stats col-lg-3 col-sm-6 col-12 py-0 my-3 my-sm-auto d-flex">
+                                <a class="mr-sm-0 mr-3 text-large w-50"
```

Comment:
It's not that important but cleaner if you use `w-25`instead of `w-50` here (as there are 4 elements) (also below, obv.)